src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -692,6 +692,13 @@ void SetGlobalPropertiesRequest::PrepareUIRequestMenuAndKeyboardData(
       cached_keyboard_props[hmi_request::auto_complete_list] =
```

Comment:
Theres a bug where if these cached values are not initialized, then it sends a setGlobalProperties with the key and a value of "undefined".

On a clean build try sending a setGlobalProperties only with an empty keyboardProperties param included.

If i send a resetGlobalProperties then everything is sent as initialized.